schemas/template_v2.json

```diff
@@ -0,0 +1,55 @@
+{
+  "$schema": "https://json-schema.org/draft/2020-12/schema",
+  "$id": "<survey-name>.json",
+  "title": "SDS schema for the <survey-name> survey",
+  "type": "object",
+  "properties": {
+    "schema_version": {
+      "const": "v2",
+      "description": "Version of the schema spec"
+    },
+    "survey_id": {
```

Comment:
The change add the survey_id as an object within properties , but what we want for survey_id is a field at the root level.

Reply:
It has been added as a top level property according to this [document](https://json-schema.org/learn/getting-started-step-by-step#define-properties:~:text=Define-,properties,-%C2%B6)